src/app/modules/main/wallet_section/send_new/view.nim

```diff
@@ -21,10 +25,42 @@ QtObject:
     result.QObject.setup
     result.delegate = delegate
     result.pathModel = newPathModel()
+    result.indexOfTxPathUnderReview = -1
 
   proc load*(self: View) =
     self.delegate.viewDidLoad()
 
+  proc getPathModel*(self: View): PathModel {.inline.} =
+    return self.pathModel
+
+  proc txPathUnderReviewChanged*(self: View) {.signal.}
```

Comment:
@saledjenic was wondering why this logic lies in nim? it can be done on the qml side, using ModelUtils and ModelEntry for example and as discussed the part that does the logic on the qml side, can be reused as a base in Send, Swap and Bridge.

Reply:
> was wondering why this logic lies in nim?

@Khushboo-dev-cpp it doesn't need to. Do you want me to move it to qml?

> can be reused as a base in Send, Swap and Bridge.

Pretty sure that either way it can.

Reply:
I think its better on the qml side. we have been trying to remove all logic from nim side as far as possible.

Rebase on top of my PR first? https://github.com/status-im/status-desktop/pull/17051 its going to be ready for merge soon.

Reply:
@Khushboo-dev-cpp updated, rebased.